paper.tex

```diff
@@ -1895,6 +1903,58 @@ \subsection{AR Art Gallery}
 	\label{fig:exhibitionScan}
 \end{figure}
 
+\subsection{Alterniative Uses for AR in ARuko}
+\label{sec:alternativeARukoAR}
+
+Even once the idea for ARuko was settled on audio tour functionality with 
+additional AR components, what exactly those AR components would do was still
+undecided for a while. Atticus knew he wanted to use AR on the tour, but wasn't
+sure what to do with it. At first, we made recommendations to him based on the
+capabilities of Vuforia that we found in our research. We knew it was good at
+image recognition and ground plane recognition, so we suggested that maybe 
+displaying some sorts of 3D models in certain locations using either of these
+features might be an interesting way to expand on the tour. With these 
+recommendations in mind, Atticus expressed interest in the capability of using
+building facades as image targets. The intention then would be to tie content
+related to a specific location within Kinkaku-ji to that physical location, and
+use images of the location as targets. As previously explained, this ultimately
+did not work out due to the limitations of image recognition with Vuforia. 
+
+After being initially deterred from image recognition, we thought that ground
+and mid-air plane recognition might the best way to implement AR in the app. 
+It didn't have to rely on images that meet Vuforia's criteria, and since it 
+could be triggered anywhere, it seemed like a good fit for an app focusing on
+walking around a site. Atticus explained that he primarily wanted to use AR to
+display art related to the site. So we focused our efforts with ground and 
+mid-air plane AR on ways to display 2D images. What we ended up creating first
+was a feature that allowed you to view certain images in AR, using ground plane
+detection, based on what region you were in. So when you entered a region of 
```

Comment:
still can't start sentence with "so" (maybe use "For example" or something else)